assets/src/apps/authoring/Authoring.tsx

```diff
@@ -161,14 +161,16 @@ const Authoring: React.FC<AuthoringProps> = (props: AuthoringProps) => {
 
   useEffect(() => {
     if (isAppVisible) {
+       // forced light mode to save on initial dev time
+      document.documentElement.classList.remove('dark');
       document.body.classList.add('overflow-hidden'); // prevents double scroll bars
     }
 
     if (!isAppVisible) {
       // reset forced light mode
       switch (getModeFromLocalStorage()) {
         case 'dark':
-          document.documentElement.classList.add('dark');
+          document.documentElement.classList.add('light');
```

Comment:
As per the requirement, the advance author will always be in '`light`' mode theme.